packages/layout/src/helpers/breakpoints.ts

```diff
@@ -0,0 +1,52 @@
+export type ResponsiveObject<T> = { sm?: T; md?: T; lg?: T; xl?: T; xxl?: T };
+export type Responsive<T> = T | ResponsiveObject<T>;
+
+export const BREAKPOINTS = ["sm", "md", "lg", "xl", "xxl"];
+export const getValueForEachBreakpoint = (
+  breakpoints: ResponsiveObject<string>,
+) => {
```

Comment:
This function gets values for each breakpoint based on the object passed in. This also fills out values for breakpoints that were not passed in. 

For example
```
{ md: "20", xxl: "2" }
```

If this object is passed in sm-md are set to `20`, and lg-xxl are set to `2`. This works the same way as min width media queries work in css.